migrations/generic/timetables/1676027954829_add_hastus_fields/up.sql

```diff
@@ -0,0 +1,52 @@
+CREATE TABLE vehicle_journey.journey_type (
+  type text PRIMARY KEY
+);
+COMMENT ON TABLE vehicle_journey.journey_type IS 'Enum table for defining allowed journey types.';
+
+INSERT INTO vehicle_journey.journey_type
+  (type)
+  VALUES
+  ('STANDARD'),
+  ('DRY_RUN'),
+  ('SERVICE_JOURNEY')
+  ON CONFLICT (type) DO NOTHING;
+
+ALTER TABLE vehicle_journey.vehicle_journey
+  ADD COLUMN journey_name_i18n jsonb,
+  ADD COLUMN turnaround_time interval,
+  ADD COLUMN layover_time interval,
+  ADD COLUMN journey_type text NOT NULL DEFAULT 'STANDARD' REFERENCES vehicle_journey.journey_type (type);
+COMMENT ON COLUMN vehicle_journey.vehicle_journey.journey_name_i18n IS 'Name that user can give to the vehicle journey.';
+COMMENT ON COLUMN vehicle_journey.vehicle_journey.turnaround_time IS 'Turnaround time. Means the time it takes to move vehicle to the starting stop and pick up the passengers.';
```

Comment:
The Transmodel definition might be a bit better: Turnaround time is the time taken by a vehicle to proceed from the end of a ROUTE to the start of another